GomokuServer/src/GomokuServer.Application/Games/Commands/Abstract/Undo.cs

```diff
@@ -0,0 +1,34 @@
+﻿
+namespace GomokuServer.Application.Games.Commands.Abstract;
+
+public abstract record UndoCommand : ICommand<UndoResponse>
+{
+	[Required]
+	public required string GameId { get; init; }
+
+	[Required]
+	public required string PlayerId { get; init; }
+}
+
+public abstract class UndoCommandHandler<TRequest>(IGamesRepository _gamesRepository)
+	: ICommandHandler<TRequest, UndoResponse>
+		where TRequest : UndoCommand
+{
+	public async Task<Result<UndoResponse>> Handle(TRequest request, CancellationToken cancellationToken)
+	{
+		var getGameResult = await _gamesRepository.GetAsync(Guid.Parse(request.GameId));
```

Comment:
Guid.parse does have fallback mechanism inside? In case if fails to parse what happens?

Reply:
If passed string is not valid Guid, it will throws exception. In our scenario it will be handled by global exceptions interceptor - ExceptionHandlingPipeline